cmd/minishift/cmd/image/cache_config_remove.go

```diff
@@ -45,15 +46,16 @@ func removeConfiguredImage(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Invalid image name: %v", err))
 	}
 
-	minishiftConfig := getMinishiftConfig()
-	cacheImages := getConfiguredCachedImages(minishiftConfig)
+	cacheImages := minishiftConfig.InstanceConfig.CacheImage
```

Comment:
TBH, I actually think variable should be named `imagesToCache := ....ImagesToCache` or similar.
`CacheImages` would sound like the images that are available inside the cache?

BUT ...
but we also gave it a second meaning, as this same name, as this variable is used to import the images to the VM from the cache`. ... think, think, think ...